reference/pbrappearance.md

```diff
@@ -0,0 +1,70 @@
+## PBRAppearance
+
+```
+PBRAppearance {
+  SFColor  baseColor            1 1 1             # any color
+  SFNode   baseColorMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  transparency         0                 # [0, 1]
+  SFFloat  roughness            0                 # [0, 1]
+  SFNode   roughnessMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  metalness            1                 # [0, 1]
+  SFNode   metalnessMap         NULL              # {ImageTexture, PROTO}
+  SFNode   environmentMap       NULL              # {Cubemap, PROTO}
+  SFFloat  IBLStrength          1                 # [0, inf)
+  SFNode   normalMap            NULL              # {ImageTexture, PROTO}
+  SFFloat  normalMapFactor      1                 # [0, inf)
+  SFNode   occlusionMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  occlusionMapStrength 1                 # [0, inf)
+  SFNode   textureTransform     NULL              # {TextureTransform, PROTO}
+  SFString name                 "PBRAppearance"   # any string
+}
+```
+
+### Description
+
+%figure "An Oxidised Copper Sphere made with the PBRAppearance node"
+
+![pbrappearance.png](images/pbrappearance.png)
+
+%end
+
+The [PBRAppearance](#pbrappearance) node specifies a a physically-based visual appearance of a node.
+The acronym "PBR" refers to "Physically-Based Rendering", a term used to designate a class of shading models based on the physical properties of an object, as opposed to approximations.
+
+This modern technique enables highly realistic rendering for every kind of object, in any kind of lighting environment.
+
+For a detailed guide on PBR and how to use it for content creation and model design, an excellent description of the concepts on this subject can be found [on this blog post by Marmoset](https://www.marmoset.co/posts/physically-based-rendering-and-you-can-too/).
+
+Our PBR shading model makes use of Lambertian Diffuse, Cook-Torrence Specular and Fresnel-Schlick GGX for direct illumination, and an advanced Image-Based Lighting (IBL) BRDF for ambient diffuse lighting and reflections.
+
+### Field Summary
+
+- The `baseColor` field specifies the base color or "albedo" of the material's surface, analogous to the `diffuseColor` field of the [Material](material.md) node.
+- The `baseColorMap` field specifies a field to use an [ImageTexture](imagetexture.md) for the material's albedo instead of a solid color.
+
+- The `transparency` field specifies the transparency of the object.
```

Comment:
Sometimes you add an empty line between the fields and sometimes not. Since this has no influence on the resulting doc we should probably be coherent.